server/chat-commands/info.ts

```diff
@@ -2521,7 +2521,8 @@ export const commands: ChatCommands = {
 		if (!/^https?:\/\//.test(link)) link = `https://${link}`;
 		link = encodeURI(link);
 		let dimensions;
-		if (!/^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.be)(\/|$)/i.test(link)) {
+		const SpotifyRegex = /open\.spotify\.com\/track\/(.*)/;
```

Comment:
See my below suggestion - just use the top level regex for this.

Reply:
Where should I define the regex?

Reply:
Put it at the top level of the file.

Reply:
Also reminder to use camelCase, so this should be `spotifyRegex`

Reply:
if it's a top level constant `SPOTIFY_REGEX` would probably be better